go.mod

```diff
@@ -10,9 +10,9 @@ require (
 	github.com/hashicorp/golang-lru/v2 v2.0.7
 	github.com/onflow/atree v0.8.0-rc.6
 	github.com/onflow/cadence v1.0.0-preview.52
-	github.com/onflow/flow-go v0.37.10
+	github.com/onflow/flow-go v0.37.10-util-ensure-checkpoint-exists.0.20240914085445-a95ba3057847
```

Comment:
Do we need that specific flow-go commit version? If so, is it deployed on testnet/mainnet?